platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileCollection.java

```diff
@@ -447,6 +448,14 @@ public synchronized List<Path> compactFiles(
                             serializationVersion, reader.readDataItemBytes(fileOffset));
                     // update the index
                     index.putIfEqual(path, dataLocation, newLocation);
+                } catch (final ClosedByInterruptException e) {
+                    logger.info(
+                            EXCEPTION.getMarker(),
```

Comment:
It should be `MERKLE_DB.getMarker()`, otherwise it will still be treated as an error (regardless of logger level) by JRS.

Reply:
Fixed